.ci/containerd_devmapper_setup.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
+
+if [ "$KATA_HYPERVISOR" != "firecracker" ]; then
+	echo "Devicemapper configuration is only for Firecracker"
+	exit
+fi
+
+sudo rm -rf /var/lib/containerd/devmapper/data-disk.img
+sudo rm -rf /var/lib/containerd/devmapper/meta-disk.img
+sudo mkdir -p /var/lib/containerd/devmapper
+sudo truncate --size 10G /var/lib/containerd/devmapper/data-disk.img
+sudo truncate --size 10G /var/lib/containerd/devmapper/meta-disk.img
+
+sudo mkdir -p /etc/systemd/system
+
+cat<<EOT | sudo tee /etc/systemd/system/containerd-devmapper.service
+[Unit]
+Description=Setup containerd devmapper device
+DefaultDependencies=no
+After=systemd-udev-settle.service
+Before=lvm2-activation-early.service
+Wants=systemd-udev-settle.service
+[Service]
+Type=oneshot
+RemainAfterExit=true
+ExecStart=-/sbin/losetup /dev/loop20 /var/lib/containerd/devmapper/data-disk.img
```

Comment:
Don't assure that whether the fixed device name will lead errors, but I suggest borrow parts of https://github.com/containerd/containerd/tree/master/snapshots/devmapper